constraints.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# Generated by first creating a virtual environment with `python3.7 -m venv .venv`, then running
+# `.venv/bin/pip install -r requirements.txt` and `.venv/bin/pip freeze`.
```

Comment:
I don't know if we should call more attention to how to create a venv? For example, should we add this to the README or make a simple script like `mk_venv.sh`?

Reply:
This is fine for now, I think.